staging/src/k8s.io/component-helpers/storage/volume/pv_helpers.go

```diff
@@ -28,8 +28,6 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	storagelisters "k8s.io/client-go/listers/storage/v1"
 	"k8s.io/client-go/tools/reference"
-	storagehelpers "k8s.io/component-helpers/storage/volume"
-	volumeutil "k8s.io/kubernetes/pkg/volume/util"
 )
 
 const (
```

Comment:
Can you rename all of these to have the prefix `Annotation`? Or maybe it's better as suffix? Have a look at what we do for other annotations.

Reply:
I found three different ways how annotations looks like:

```sh
grep -r --include "*.go" --exclude-dir="vendor" -E "\sAnnotation[[:alpha:]]{2,} = " . | wc -l
10

grep -r --include "*.go" --exclude-dir="vendor" -E "\s[[:alpha:]]{2,}AnnotationKey = " . | wc -l
26

grep -r --include "*.go" --exclude-dir="vendor" -E "\s[[:alpha:]]{2,}Annotation = " . | wc -l
45
```

@alculquicondor What do you think If I suffix it with `Annotation`? For example, `AnnBindCompleted` will be `BindCompletedAnnotation`.

Reply:
There are too many changes:
```
        modified:   pkg/controller/volume/persistentvolume/binder_test.go
        modified:   pkg/controller/volume/persistentvolume/delete_test.go
        modified:   pkg/controller/volume/persistentvolume/framework_test.go
        modified:   pkg/controller/volume/persistentvolume/provision_test.go
        modified:   pkg/controller/volume/persistentvolume/pv_controller.go
        modified:   pkg/controller/volume/persistentvolume/pv_controller_base.go
        modified:   pkg/controller/volume/persistentvolume/pv_controller_test.go
        modified:   pkg/controller/volume/persistentvolume/recycle_test.go
        modified:   pkg/scheduler/framework/plugins/volumebinding/assume_cache_test.go
        modified:   pkg/scheduler/framework/plugins/volumebinding/binder.go
        modified:   pkg/scheduler/framework/plugins/volumebinding/binder_test.go
        modified:   pkg/scheduler/framework/plugins/volumebinding/test_utils.go
        modified:   pkg/scheduler/generic_scheduler_test.go
        modified:   plugin/pkg/admission/storage/persistentvolume/label/admission.go
        modified:   plugin/pkg/admission/storage/persistentvolume/label/admission_test.go
        modified:   staging/src/k8s.io/component-helpers/storage/volume/pv_helpers.go
        modified:   test/integration/scheduler_perf/scheduler_perf_legacy_test.go
        modified:   test/integration/util/util.go
        modified:   test/integration/volume/persistent_volumes_test.go
```
@alculquicondor I think, Its's better to do this in separate PR.  What do you think?

Reply:
As long as it happens before the 1.24 code freeze, fine to do in a follow up PR.